src/platform/qt/ShortcutController.cpp

```diff
@@ -555,8 +552,8 @@ ShortcutController::ShortcutItem::ShortcutItem(QAction* action, const QString& n
 ShortcutController::ShortcutItem::ShortcutItem(ShortcutController::ShortcutItem::Functions functions, int shortcut, const QString& visibleName, const QString& name, ShortcutItem* parent)
 	: m_action(nullptr)
```

Comment:
I'm a dingus and missed these (will address alongside everything else that gets flagged down, if any).